source/js/main.js

```diff
@@ -186,6 +190,19 @@ let main = {
     });
   },
 
+  bindGAEventTrackers() {
+    document.querySelector(`#see-more-modular-page`).addEventListener(`click`, () => {
+      let label = ``;
+      let pageHeader = document.querySelectorAll(`.cms h1`)[0];
```

Comment:
You can just use `querySelector` if you only want the first result.

Reply:
oh yea apparently i wasn't thinking 🤦‍♂️